eclass/kernel-build.eclass

```diff
@@ -213,6 +206,32 @@ kernel-build_src_install() {
 	dosym ../../../usr/src/linux-${ver} /lib/modules/${ver}/build
 	dosym ../../../usr/src/linux-${ver} /lib/modules/${ver}/source
 
+	# Install the unstripped uncompressed vmlinux for use with systemtap
+	# etc.  Use mv rather than doins for the same reason as above --
+	# space and time.
+	if use debug; then
+		local vmlinux_path="build/vmlinux"
+		local debug_path="/usr/lib/debug"
+		debug_path="${debug_path}/lib/modules/${ver}"
+
+		if ! has splitdebug ${FEATURES}; then
```

Comment:
Ok, this is a no-go. FEATURES is sekrit PM variable that's not for us mortals to touch. (yes, I know many ebuilds/eclasses already suck at this but that's no excuse to make things worse)

How about we decide to either do it one way or another, and have Portage effectively ignore the file via `dostrip -x`? Would that work?

Reply:
Oh, I did not know, thanks for pointing that out. However I'm not sure what you mean exactly. Do you mean to just pick whether to split the debuginfo or not and then just do `dostrip -x`? I'm fine with that, I just wanted to honour as much user settings as possible.

So if this is the case then I'd rather pick the whole vmlinux since the debug symbols do not take as much space and the unstripped debug binary is almost as huge as the original one and for ppc64 we're not stripping it either, so at least on ppc64 that would save disk space since we wouldn't need to install both `vmlinux` *and* `vmlinux.debug`.

I'll fix these in a bit, thanks for the review.